upgrading/automated_upgrades.adoc

```diff
@@ -872,6 +872,23 @@ You can set the `openshift_logging_elasticsearch_replace_configmap` parameter to
 `logging-elasticsearch` ConfigMap  with the current default values. In some cases, using an older
 ConfigMap can cause the upgrade to fail. The default is set to `false`. For more information, see the parameter
 in xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[specify logging Ansible variables].
++
+[NOTE]
+====
+If your Fluentd `DeploymentConfig` and `DaemonSet` for the EFK components are
```

Comment:
As you mentioned in slack, I don't think these are marked up in 3.11. I'd consider adding "objects" after the two though.

Reply:
@bergerhoffer 
I didn't find any references to _`DeploymentConfig` object_ or _`DaemonSet` object_ in 3.11. I did just find some _`*DeploymentConfig*` object_ and _`*DaemonSet*` objects_. Quite inconsistent are these docs!